assets/src/dashboard/animations/animationOutputs/keyframesOutput.js

```diff
@@ -24,7 +24,7 @@ import PropTypes from 'prop-types';
  */
 import { GeneralAnimationPropTypes, KeyframesPropTypes } from './types';
 
-function AnimationOutput({ id, keyframes, ...options }) {
+function KeyframesOutput({ id, keyframes, ...options }) {
```

Comment:
I renamed `AnimationOutput` -> `KeyframesOutput`.  I feel the new name better describes what it does, but this resulted in lots of files needing a quick name update (bloating this PR 😱 )